app/src/main/java/mozilla/lockbox/action/TelemetryAction.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.action
+
+import mozilla.lockbox.flux.Action
+import org.mozilla.telemetry.event.TelemetryEvent
+
+open class TelemetryAction(
```

Comment:
Is there a reason this is a `class` and not an `interface`? I'd love to see an example of how this will be implemented on individual `Action`s in this PR.

Reply:
This was copied almost exactly from iOS.  I don't see a good reason to make this an interface devoid of implementation right now.

Reply:
IRL conversation ... make `TelemetryAction` abstract (and vals abstract) so it cannot be used standalone.